src/Root/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from '../App';
+import { HomePage } from '../pages/HomePage';
+import { PeoplePage } from '../pages/PeoplePage';
+import { NotFoundPage } from '../pages/NotFoundPage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
```

Comment:
The best practice for preventing mistakes in the future is to create variables for paths like "/". Create variables and put them into a separate file.